cloudinit/net/freebsd.py

```diff
@@ -50,10 +51,8 @@ def start_services(self, run=False):
         for dhcp_interface in self.dhcp_interfaces():
             # Observed on DragonFlyBSD 6. If we use the "restart" parameter,
             # the routes are not recreated.
-            subp.subp(
-                ["service", "dhclient", "stop", dhcp_interface],
-                rcs=[0, 1],
-                capture=True,
+            net.dhcp.IscDhclient.start_service(
```

Comment:
@holmanb it looks like this PR unintentionally inverts the operations, the intent here was to stop any existing dhcp services on any dhcp interfaces before starting them again (instead of just using the systemctl restart operation). But it looks like we are now starting and then stopping. Let's flip them back.

Reply:
Nice catch. Fixed.